ts/src/pro/kraken.ts

```diff
@@ -483,26 +481,64 @@ export default class kraken extends krakenRest {
+        /**
+         * @method
+         * @name kraken#watchTickers
+         * @description watches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+         * @param {string} symbol unified symbol of the market to fetch the ticker for
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object} a [ticker structure]{@link https://docs.ccxt.com/#/?id=ticker-structure}
+         */
+        await this.loadMarkets ();
+        symbols = this.marketSymbols (symbols, undefined, false);
+        const ticker = await this.watchMultiHelper ('ticker', 'ticker', symbols, undefined, params);
+        if (this.newUpdates) {
+            const result = {};
+            result[ticker['symbol']] = ticker;
+            return result;
+        }
+        return this.filterByArray (this.tickers, 'symbol', symbols);
     }
 
     async watchTrades (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Trade[]> {
         /**
          * @method
          * @name kraken#watchTrades
          * @description get the list of most recent trades for a particular symbol
+         * @see https://docs.kraken.com/websockets/#message-trade
          * @param {string} symbol unified symbol of the market to fetch trades for
          * @param {int} [since] timestamp in ms of the earliest trade to fetch
          * @param {int} [limit] the maximum amount of trades to fetch
          * @param {object} [params] extra parameters specific to the exchange API endpoint
          * @returns {object[]} a list of [trade structures]{@link https://docs.ccxt.com/#/?id=public-trades}
          */
-        await this.loadMarkets ();
-        symbol = this.symbol (symbol);
-        const name = 'trade';
-        const trades = await this.watchPublic (name, symbol, params);
+        return await this.watchTradesForSymbols ([ symbol ], since, limit, params);
+    }
+
+    async watchTradesForSymbols (symbols: string[], since: Int = undefined, limit: Int = undefined, params = {}): Promise<Trade[]> {
+        /**
+         * @method
+         * @name kraken#watchTradesForSymbols
+         * @see https://docs.kraken.com/websockets/#message-trade
+         * @description get the list of most recent trades for a list of symbols
+         * @param {string[]} symbols unified symbol of the market to fetch trades for
+         * @param {int} [since] timestamp in ms of the earliest trade to fetch
+         * @param {int} [limit] the maximum amount of trades to fetch
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @returns {object[]} a list of [trade structures]{@link https://docs.ccxt.com/#/?id=public-trades}
+         */
+        const trades = await this.watchMultiHelper ('trade', 'trade', symbols, { 'since': since, 'limit': limit }, params);
```

Comment:
why are we providing since/limit to watchMultipleHelper? that does not seem to make sense

Reply:
I have started to do with all methods, specifically at this moment and specifically for the `watchTrades` it is not needed (but needed for watchOb's and I have to update that in other exchanges too), however, i think it's not a bad practice to store the provided user-arguments within subscriptions, it will not harm, so in later stages, we would be able to move out: 
```
            const first = this.safeList (trades, 0);
            const tradeSymbol = this.safeString (first, 'symbol');
            limit = trades.getLimit (tradeSymbol, limit);
```
from `watchTrades` individual implementations from, because `limit` argument is only accessible inside `watchTrades` atm, so i was thinking about long-term/better approach

Reply:
so, the effect is not immediate, but would make sense if you approve doing similarly on other exchanges too and for any `watch` methods (because in near future we would also need that for `subscribe` ED approach) and this approach wouldn't harm if we start doing them from now in any watch methods

Reply:
you can see how we can access that later (atm such can be seen in coinbase, binance ...): https://github.com/ccxt/ccxt/blob/master/ts/src/pro/coinbase.ts#L518

Reply:
the orderbook is different, the limit might be important to set up the cache, but for watchTrades we don't need to provide since/limit and we don't need it for the subscribe either (that filtering can be done before returning to the user)

Reply:
ok removed & pushed